diff --git a/class.c b/class.c
index 7e1ba13..a082c22 100644
--- a/class.c
+++ b/class.c
@@ -1,11 +1,8 @@
-#include <stdio.h>
+#include <stdio.h> // add comment here
 #include <stdlib.h> // malloc
 
-#include "class.h"
-
 struct _Private {
 	char _readBuf[READ_BUF_SIZE];
-	char* _begin;
 };
 
 static int push(Class* obj, const char c)
@@ -15,21 +12,27 @@ static int push(Class* obj, const char c)
 
 static int peep(const Class* obj, char* c)
 {
-	return 0;
+	int i = 0;
+	i++;
+	return i;
 }
 
-
+// change empty line to a comment
 Class* new_Class() {
 	Class* obj = (Class*) malloc(sizeof(Class));
-	//obj->_begin = obj->_readBuf;
-	//*(obj->_begin) = 0;
-	obj->push = push;
-	obj->peep = peep;
+
+	obj->_begin = obj->_readBuf;
+	*(obj->_begin) = 0;
+
 	return obj;
 }
 
 void delete_Class(Class* obj) {
+	int i = 0;
+	int j = 2;
+	int k = 3;
 	free(obj);
 	obj = NULL;
+	return;
 }
 
